comit/Cargo.toml

```diff
@@ -34,6 +34,9 @@ serdebug = "1"
 sha2 = "0.9"
 strum = "0.18.0"
 strum_macros = "0.18.0"
+# dev-dependencies can't be optional :(
```

Comment:
`can't be optional` but `optional=true`? I don't understand :)

Reply:
Read the comment again :)

"dev-dependencies can't be optional" - hence this dependency is under `[dependencies]` although it is just used for tests ;)

Reply:
aahhh... got it :D